layouts/services/course.html

```diff
@@ -37,6 +35,12 @@ <h4>{{ i18n "translations" }}</h4>
 
   {{ partial "related" . }}
 
+  <br>
+
+  {{ partial "related-form" . }}
+
+  {{ partial "course/grades" . }}
```

Comment:
Maybe we can have the form come after grades? 

Since grades doesn't work right now, there is no need to change at the moment but we can probably change it after we fix the grade api call issue